.github/workflows/cibuildwheel.yml

```diff
@@ -41,6 +41,8 @@ jobs:
       - name: Build wheels
```

Comment:
I think you'll need a step just before the "Build wheels" step which sets up QEMU for emulation:

```yaml
      - name: Set up QEMU
        if: runner.os == 'Linux'
        uses: docker/setup-qemu-action@v2
        with:
          platforms: arm64
```

Reply:
Right you are! Thanks.